miasm2/analysis/dse.py

```diff
@@ -215,7 +218,7 @@ def breakpoint(self, jitter):
         self.prepare()
 
     def handle(self, cur_addr):
-        """Handle destination
+        r"""Handle destination
         @cur_addr: Expr of the next address in concrete execution
         /!\ cur_addr may be a lbl_gen
```

Comment:
`cur_addr` may not be anymore a "lbl_gen" (ie. the comment doesn't stand anymore). But it can be a LocKey

Reply:
done